wordlist.txt

```diff
@@ -60,3 +60,4 @@ VNET
 TKG
 NSXT
 autoscaler
+airgap
```

Comment:
Please change to "air-gapped"

Reply:
Craig, airgap without the hypen is good.